deps/rabbit/src/tcp_listener_sup.erl

```diff
@@ -52,3 +54,8 @@ init({IPAddress, Port, Transport, SocketOpts, ProtoSup, ProtoOpts, OnStartup, On
         Transport, RanchListenerOpts,
         ProtoSup, ProtoOpts),
     {ok, {Flags, [RanchChildSpec, OurChildSpec]}}.
+
+max_conn(infinity, _) ->
+    infinity;
+max_conn(Max, Sups) ->
+    Max * Sups.
```

Comment:
You probably want `Max div Sups` here instead? Otherwise if I have `connection_max` set to 1000 and I have 1000 `conns_sups` then the Ranch `max_connections` will be 1 million which is per `conns_sup`, so the real max will end up being 1 billion. :-)